### ISSUE TITLE
Add support for the Levoit Dual 200S humidifier

### ISSUE THREAD

**Issue by unholyhimura** [formatBCE/HA_VeSync_Classic300s#1]
Hello @formatBCE, I'm trying to add support to Levoit Dual 200s Humidifier from your code. 

From another user: "The Dual 200S humidifer seems identical to the Classic 300S humidifier, except it has no night-light."

Quick test on my side, I've modified common.py file by replacing:
```
HUMI_DEV_TYPE_TO_HA = {
    "Classic300S": "humidifier"
} 
```
to

```
HUMI_DEV_TYPE_TO_HA = {
    "Dual200S": "humidifier"
}
```

![image](https://user-images.githubusercontent.com/6892943/143953595-b331ed6d-af7b-4bb1-a67e-98f33bf9f3dc.png)

And I've modified humidifier.py file by replacing:

```
PRESET_MODES = {
    "Classic300S": [MODE_AUTO, MODE_SLEEP, MANUAL_LOW, MANUAL_MID, MANUAL_HIGH],
}
```
to

```
PRESET_MODES = {
    "Dual200S": [MODE_AUTO, MODE_SLEEP, MANUAL_LOW, MANUAL_MID, MANUAL_HIGH],
}
```

My fix didn't work, the humidifier is not appearing in Home Assitant, it's not that simple? I don't get any errors from Home Assistant Logs. 

Thanks for any help :)

**Comment by formatBCE** [comment]
Hello @unholyhimura!

Case is, that this integration relies on pyvesync library.
https://github.com/webdjoe/pyvesync

So if that library doesn't support your device, this integration won't as well.

I did not find any mentions of 200S Humidifier there (there are Core 200s purifier though, so it might get you confused).
Levoit has no imagination in devices naming, so one need pay close attention while searching.

Anyway, I can add support for your device only after pyvesync has methods to work with it.
Cheers.

**Comment by formatBCE** [comment]
I added support for Dual200S

**Comment by maikeruk** [comment]
Hello @formatBCE  thanks a lot for your great work. I do have the same system but it is not found by the home assistant integration is there anything in particular that I have to do? sorry for the noobie question.

**Comment by formatBCE** [comment]
> Hello @formatBCE  thanks a lot for your great work. I do have the same system but it is not found by the home assistant integration is there anything in particular that I have to do? sorry for the noobie question.

Hi! 
First of all, you can use pyvesync library directly to check, if it does see your device and can manipulate it.
If it does, then we'll try to fix the issue :)

**Comment by davideschito** [comment]
Hello, I have Dual200s which is connected to the network and works in VeSync app.
I have installed your custom integration but it does not see any devices.
I tried with pyvesync, it logs but when I call get_devices() it gives me "True" and that's all.
Any suggestions?
Thanks!!

**Comment by formatBCE** [comment]
Hi! I don't really remember, how to work with pyvesync, but looks like you need to check some array field after running get_devices - the command itself won't output data. Like manager.fans[0] something like this.

**Comment by davideschito** [comment]
Thank you, I was able to communicate with the device after installing the version of the integration that supported v1.4.3 of pyvesync.
________________________________
Da: formatBCE ***@***.***>
Inviato: Monday, August 8, 2022 7:43:26 PM
A: formatBCE/HA_VeSync_Classic300s ***@***.***>
Cc: davideschito ***@***.***>; Comment ***@***.***>
Oggetto: Re: [formatBCE/HA_VeSync_Classic300s] Add support for the Levoit Dual 200S humidifier (Issue #1)


Hi! I don't really remember, how to work with pyvesync, but looks like you need to check some array field after running get_devices - the command itself won't output data. Like manager.fans[0] something like this.

**Comment by jacksnchz** [comment]
Hi all,

Sorry opening the issue again...recently I installed the integration (VeSync-FormatBCE v1.0.4) but no way to recognize the device, I removed and installed the integration with and without homeassistant restart without success.

Logs reports me next information:

`2024-08-28 12:26:48.626 WARNING (MainThread) [custom_components.vesync_formatbce.fan] Humidificador - Unknown device type - LUH-D301S-WEU`

Humidifier is working currently via manufacter app and google home.

No cloud integration installed, only this one.

Thank u!
